config.sample.yaml

```diff
@@ -27,7 +27,7 @@ homeserver:
   # is the "domain name" part of the HS URL.
   domain: "localhost"
 
-  # Should presence be enabledfor matrix clients on this bridge. If disabled on the 
+  # Should presence be enabledfor matrix clients on this bridge. If disabled on the
```

Comment:
enabledfor

Reply:
Historical spelling errors are the worst :(